theano/gpuarray/opt.py

```diff
@@ -1092,6 +1093,13 @@ def local_gpua_advanced_incsubtensor(op, context_name, inputs, outputs):
             set_instead_of_inc=set_instead_of_inc)
 
 
+@register_opt('fast_compile')
+@op_lifter([tensor.AdvancedIncSubtensor])
+@register_opt2([tensor.AdvancedIncSubtensor], 'fast_compile')
+def local_gpua_advanced_incsubtensor(op, context_name, inputs, outputs):
+    return GpuAdvancedIncSubtensor()
```

Comment:
That's not correct, we should only do that if `op.set_instead_of_inc` is False!
Otherwise, we would be replacing set_subtensor with inc_subtensor.

Later, it would be nice to ad the `set_instead_of_inc` arg to the GPU Op as well.